doandroidlib/src/main/java/in/tosc/doandroidlib/api/DigitalOceanClient.java

```diff
@@ -49,6 +51,10 @@
             @Field("name") String name              // for rename or snapshot
             );
```

Comment:
Please indent this piece.
It's a good practice to fix any indentation issues that exist in the files which you are committing.